components/raftstore-v2/src/raft/apply.rs

```diff
@@ -64,6 +64,7 @@ pub struct Apply<EK: KvEngine, R> {
     admin_cmd_result: Vec<AdminCmdResult>,
     flush_state: Arc<FlushState>,
     sst_apply_state: SstApplyState,
+    pub(crate) sst_applied_index: Vec<SstApplyIndex>,
```

Comment:
Please use getter and setter to keep it consistent.

Reply:
done